subprojects/veins_testsims/src/veins_testsims/traci/TraCITestApp.cc

```diff
@@ -205,6 +205,16 @@ void TraCITestApp::handlePositionUpdate()
         }
     }
 
+    if (testNumber == testCounter++) {
+        if (t == 1) {
+            assertEqual("(TraCICommandInterface::getVehicleTypeMaxSpeed speed is correct)", traci->getVehicleTypeMaxSpeed("vtype0"), 70);
+            traci->setVehicleTypeMaxSpeed("vtype0", 60);
+            assertEqual("(TraCICommandInterface::getVehicleTypeMaxSpeed) changed speed is correct", traci->getVehicleTypeMaxSpeed("vtype0"), 60);
```

Comment:
this seems to test `set...` rather than `get...`, so the comment should be updated I guess